packages/manager/modules/pci/src/projects/project/data-processing/notebooks/terminate-notebook/terminate-notebook.controller.js

```diff
@@ -0,0 +1,19 @@
+export default class {
+  /* @ngInject */
+  constructor(dataProcessingService) {
+    this.dataProcessingService = dataProcessingService;
+    this.stopNotebook = this.stopNotebook.bind(this);
```

Comment:
**question:** any reason for doing this?

Reply:
I used old data-processing pages as a base and did not remove it when I should have. Fixed :)